cmd/appgw-ingress/main.go

```diff
@@ -45,12 +45,13 @@ const (
 )
 
 var (
-	flags          = pflag.NewFlagSet(`appgw-ingress`, pflag.ExitOnError)
-	inCluster      = flags.Bool("in-cluster", true, "If running in a Kubernetes cluster, use the pod secrets for creating a Kubernetes client. Optional.")
-	kubeConfigFile = flags.String("kubeconfig", "", "Path to kubeconfig file with authorization and master location information.")
-	resyncPeriod   = flags.Duration("sync-period", resyncPause, "Interval at which to re-list and confirm cloud resources.")
-	versionInfo    = flags.Bool("version", false, "Print version")
-	verbosity      = flags.Int(verbosityFlag, 1, "Set logging verbosity level")
+	flags                 = pflag.NewFlagSet(`appgw-ingress`, pflag.ExitOnError)
+	inCluster             = flags.Bool("in-cluster", true, "If running in a Kubernetes cluster, use the pod secrets for creating a Kubernetes client. Optional.")
+	kubeConfigFile        = flags.String("kubeconfig", "", "Path to kubeconfig file with authorization and master location information.")
+	resyncPeriod          = flags.Duration("sync-period", resyncPause, "Interval at which to re-list and confirm cloud resources.")
+	versionInfo           = flags.Bool("version", false, "Print version")
+	verbosity             = flags.Int(verbosityFlag, 1, "Set logging verbosity level")
+	allowSystemNamespaces = flags.Bool("allow-system-namespaces", false, "Resources in kube-system and kube-public will not be ignored")
```

Comment:
Can you change this to an environment variable instead ?
Also, you will need to modify the helm templates so that the setting flows from helm values to the deployment.

You can refer to this [commit](https://github.com/Azure/application-gateway-kubernetes-ingress/commit/093de881fb9abba2243d85a8bcba2cb853614ef7) as a reference.